backtesting/test/_test.py

```diff
@@ -642,6 +642,14 @@ def test_params(self):
                 with self.subTest(param=p[0]):
                     bt.plot(**dict([p]), filename=f, open_browser=False)
 
+    def test_hide_legend(self):
+        bt = Backtest(GOOG.iloc[:100], SmaCross)
+        bt.run()
+        with _tempfile() as f:
+            bt.plot(filename=f, show_legend=False)
+            # Give browser time to open before tempfile is removed
+            time.sleep(10)
```

Comment:
```suggestion
            time.sleep(5)
```
Waiting is bad enough as it is.